src/tools.ts

```diff
@@ -81,7 +81,11 @@ export class ToolsConfig {
                     } while (action === 'Download again');
 
                     if (action !== 'Cancel') {
-                        await Archive.unzip(toolDlLocation, path.resolve(Platform.getUserHomePath(), '.vs-tekton'), "");
+                        if (toolDlLocation.endsWith('.zip') || toolDlLocation.endsWith('.tar.gz')) {
+                            await Archive.unzip(toolDlLocation, path.resolve(Platform.getUserHomePath(), '.vs-tekton'), ToolsConfig.tool['tkn'].filePrefix);
```

Comment:
This will not work, because Archive.uinzip have no idea how to deal with zip and gz files.